sort-usermap.js

```diff
@@ -1,13 +1,15 @@
 const fs = require('fs');
+const _ = require('lodash');
```

Comment:
I don't think we have lodash dependency and it's an overkill to include that just for this script.

If enforcing uniqueness is your target, you can use ES6 Set.

Reply:
There's a transitive dependency to lodash, but I can see why this can be an overkill. Will use set then!